### DISCUSSION TITLE
Hiding buttons on the left menu

### DISCUSSION THREAD

**Discussion by mustafacanp** [epam/ketcher#710] [General]
I want to hide all buttons below the "Rotate Tool" on the left menu. Do you have a plan to add these buttons to the `buttons` property of the `Editor` component?

e.g
```
<Editor
    buttons={{
        sgroup: {hidden: true},
        datasgroup: {hidden: true},
        ...
    }}
/>
```

![Screenshot 2021-08-09 162058](https://user-images.githubusercontent.com/16281650/128720858-f01866fb-25c2-4b09-a75e-9b358a4fd2fd.png)

**Comment by AndreiMazol** [comment]
Hi @mustafacanp ,

we will extend the functionality of hiding buttons in this release. The task has been created https://github.com/epam/ketcher/issues/713. 

Thank you  for your ideas.

Best regards,
Andrei

**Reply by AndreiMazol** [reply]
Hi @mustafacanp ,

The issue is done, please check if implemented approach is suitable for you (master branch)

Best regards,
Andrei

**Reply by mustafacanp** [reply]
Thank you! Is that fixed on the 2.3.1 branch?

**Reply by AndreiMazol** [reply]
Unfortunately, no. This feature will bi delivered as a part of release 2.4, so  this moment it exists in master branch only.

**Comment by robtree** [comment]
Hello! Will button-hiding configuration be exclusive to the React plugin, or might it be possible to configure button visibility as we do when configuring the API path for the IFRAME src via, eg. `index.html?api_path=${API}/ketcher`?

**Reply by AndreiMazol** [reply]
Hi @robtree ,

buttons visibility will be managed by passing a special parameter, <Editor> -> 'buttons' property. Reading of query parameters is a responsibility of an application that uses ketcher-react library, so if you want we can extend our example app (it is delivered as *.zip) to allow it to initialize <Editor> dynamically.

**Reply by robtree** [reply]
We're using Ketcher in a Vue.js application by unzipping ketcher-standalone-2.3.1.zip into the public folder (https://cli.vuejs.org/guide/html-and-static-assets.html#the-public-folder) and then loading `<iframe src="/ketcher/index.html?api_path=/api/ketcher"></iframe>` in a persistent Vue component. If there's a way to configure the buttons without generating a custom build of Ketcher, that'd be ideal.

**Reply by AndreiMazol** [reply]
Dear @robtree ,

sorry for delay. Your request will be implemented in scope of https://github.com/epam/ketcher/issues/862

**Comment by hellhorse123** [comment]
Hello, Is there a functionality to hide the toolbar? In particular, is it possible to hide the toolbar part?
